assets/app/vue/views/MailView/sections/DashboardSection/components/ViewServerSettings.vue

```diff
@@ -0,0 +1,277 @@
+<script setup lang="ts">
+import { ref, computed } from 'vue';
+import { useI18n } from 'vue-i18n';
+import { PhSliders, PhGlobe, PhCopySimple, PhKey, PhInfo } from '@phosphor-icons/vue';
+import DetailsSummary from '@/components/DetailsSummary.vue';
+
+const { t } = useI18n();
+
+enum INCOMING_SERVER_TABS {
+  IMAP = 'imap',
+  JMAP = 'jmap',
+}
+
+const incomingServerSelectedTab = ref<INCOMING_SERVER_TABS>(INCOMING_SERVER_TABS.IMAP);
+
+const imapServerDetails = {
+  protocol: 'IMAP',
+  hostname: 'imap.thundermail.com',
+  domain: 'thundermail.com',
+  port: 993,
+  authentication: 'SSL/TLS',
+};
+
+const jmapServerDetails = {
+  protocol: 'JMAP',
+  hostname: 'jmap.thundermail.com',
+  domain: 'thundermail.com',
+  port: 443,
+  authentication: 'SSL/TLS',
+}
+
+const smtpServerDetails = {
+  protocol: 'SMTP',
+  hostname: 'smtp.thundermail.com',
+  domain: 'thundermail.com',
+  port: 993,
+  authentication: 'SSL/TLS',
+}
+
+const incomingServerDetails = computed(() => {
+  return incomingServerSelectedTab.value === INCOMING_SERVER_TABS.IMAP ? imapServerDetails : jmapServerDetails;
+});
+
+const copyValue = async (value: string | number) => {
+  try {
+    await navigator.clipboard.writeText(String(value));
+  } catch (_err) {
+    // noop
```

Comment:
Wanna cut a ticket to show an error message? I don't think we have a toast notification component or pattern setup to do this atm.